Rover/Parameters.h

```diff
@@ -433,6 +433,8 @@ class ParametersG2 {
 
     // manual mode steering expo
     AP_Float manual_steering_expo;
+
+    AP_Float fs_gcs_timeout;
```

Comment:
let's put a little comment above this parameter so it matches the others.  I know that its purpose is pretty clear from the name but..

Reply:
done